src/NSubstitute/Substitute.cs

```diff
@@ -89,4 +90,19 @@ public static T ForPartsOf<T>(params object[] constructorArguments)
         var substituteFactory = SubstitutionContext.Current.SubstituteFactory;
         return (T)substituteFactory.CreatePartial([typeof(T)], constructorArguments);
     }
+
+        public static T ForTypeForwardingTo<T,TClass>(params object[] constructorArguments)
```

Comment:
**chore:** could you please add doc comments here (see `ForPartsOf` and other public methods for examples).